### ISSUE TITLE
Wrong format in python plugin configuration

### ISSUE THREAD

**Issue by ttarczynski** [pdxcat/puppet-module-collectd#281]
## Issue description

After upgrading to pdxcat/collectd 3.4.0 I get wrong formatting in the python plugin configuration.

* puppet code:

```
    collectd::plugin::python::module {'rabbitmq':
      modulepath    => '/usr/lib64/collectd',
      script_source => 'file:///usr/lib64/collectd/rabbitmq.py',
      config        => {
        'Username'    => 'uuu',
        'Password'    => 'ppp',
        'Realm'       => 'RabbitMQ Management',
        'Host'        => 'localhost',
        'Port'        => '15672',
        'PrefixVhost' => true,
      }
    }
```

* generated configuration file:

```
# cat /etc/collectd.d/python-config.conf 
# Generated by Puppet
<Plugin "python">
  ModulePath "/usr/lib64/collectd"
  LogTraces false
  Interactive false

  Import "rabbitmq"

  <Module "rabbitmq">
      Host localhost      Password ppp      Port 15672      PrefixVhost true      Realm RabbitMQ Management      Username uuu  
  </Module>

</Plugin>
```

* I think the `<Module "rabbitmq">` block should be formatted like this:

```
  <Module "rabbitmq">
      Host "localhost"    
      Password "ppp"
      Port "15672"    
      PrefixVhost true
      Realm "RabbitMQ Management"
      Username "uuu"
  </Module>
```
## Environment

```
# puppet --version
3.7.4
# ruby -v
ruby 1.8.7 (2013-06-27 patchlevel 374) [x86_64-linux]
# cat /etc/redhat-release 
CentOS release 6.6 (Final)
```

## Suggested cahnges

* add new line after each param
* add double quotes for values that are strings (but not for numbers and booleans)

---

I will try to submit a PR with this change.

**Comment by ttarczynski** [comment]
I've submitted PR #283 to add missing newlines.
And I've learned I can get double quotes with the following puppet code:

```
    collectd::plugin::python::module {'rabbitmq':
      modulepath    => '/usr/lib64/collectd',
      script_source => 'file:///usr/lib64/collectd/rabbitmq.py',
      config        => {
        'Username'    => '"uuu"',
        'Password'    => '"ppp"',
        'Realm'       => '"RabbitMQ Management"',
        'Host'        => '"localhost"',
        'Port'        => '"15672"',
        'PrefixVhost' => 'true',
      }
    }
```

**Comment by blkperl** [comment]
Fixed in v4.0.0
